prestopantry_app/tests/view_tests/test_my_pantry_page_view.py

```diff
@@ -0,0 +1,37 @@
+from django.test import TestCase
+from django.urls import reverse
+from prestopantry_app.models.users import User
+from prestopantry_app.models.user_ingredients import UserIngredient
+
+
+class MyPantryPageViewTest(TestCase):
+    def setUp(self):
+        self.user = User.objects.create_user('goldmember','drevil@gmail.com','ilovegold')
+
+    def test_my_pantry_view(self):
+        self.client.force_login(self.user)
+        response = self.client.get('/pantry/')
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'pantry.html')
+
+    def test_my_pantry_ingredients(self):
+         # login
+        self.client.force_login(self.user)
+        # go to pantry search page
+        response = self.client.get(reverse('pantry'))
+        self.assertEqual(response.status_code, 200)
+        self.assertTemplateUsed(response, 'pantry.html')
+        self.assertContains(response, '<h2> My Pantry </h2>')
```

Comment:
Redundant, `assertTemplateUsed` is doing the same thing. (i had adam remove it from the test he had it in aswell)